jetty-start/src/main/java/org/eclipse/jetty/start/StartArgs.java

```diff
@@ -1066,6 +1072,14 @@ public void parse(final String rawarg, String source)
             return;
         }
 
+        // to override default http://central.maven.org/maven2/
+        if (arg.startsWith("--maven-base-url="))
```

Comment:
Why a new option and not just a property like we do with `jetty.home` and `jetty.base`?  It could be:
```sh
java -jar start.jar maven.base=file:///opt/mylocalmaven/ --add-to-start=foobar
```
or/and
```sh
java -Dmaven.base=file:///opt/mylocalmaven/ -jar start.jar  --add-to-start=foobar
```